.golangci.yaml

```diff
@@ -32,7 +33,10 @@ linters-settings:
   gofumpt:
     module-path: github.com/databricks/cli
     extra-rules: true
-  #goimports:
-  #  local-prefixes: github.com/databricks/cli
+  testifylint:
+    enable-all: true
+    disable:
+      # good check, but we have too many assert.(No)?Errorf? so excluding for now
+      - require-error
```

Comment:
What does it do?

Reply:
It tells you to use require.NoError(t, err) instead of assert.NoError(t, err). Presumably because otherwise you're reading bad values in the subsequent code.